app/widgets/cell.py

```
@@ -67,6 +364,17 @@ def __init__(self, cell_type="Code"):
         self.anim_timer.start(60)
         self.anim_phase = 0
 
+    def open_debugger(self):
+        """Open the interactive debugger for this code cell (Code cells only)."""
+        if self.cell_type.currentText() != "Code":
+            return
+        code = self.input.toPlainText()
+        lines = code.splitlines()
+        if not any(l.strip() for l in lines):
```

Ruff (0.15.7)

[error] Ambiguous variable name: `l`

(E741)